apluslms_roman/validator.py

```diff
@@ -0,0 +1,85 @@
+import yaml
+from jsonschema import validate
+import re
+import os
+#from operator import itemgetter
+import logging
+
+logger = logging.getLogger(__name__)
+
+#mystinen lista schema directoryja - course.yamlissa jossain kentässä, joka sijaitsee käyttäjän paketin juuressa
+def get_folders(config):
+    folders = config.__config__.get("custom_schema_locations", [])
```

Comment:
Key `schema_paths` should be semantically better. In addition, it should be validated that it's a list.

```
if not isinstance(folders, list):
  raise ValueError("Invalid type for 'schema_path' in config. Expected a llist.")
```